azure-pipelines.yml

```diff
@@ -0,0 +1,33 @@
+pr: none
+trigger:
+  batch: true
+  branches:
+    include:
+      - '*'
+
+pool:
+  vmImage: ubuntu-latest
+
+variables:
+  DOCKER_BUILDKIT: 1
+  imageName: 'keboola/billing-api-php-client'
+
+stages:
+  - stage: tests
+    displayName: Build & Test
+    jobs:
+      - job: test
+        displayName: 'Test'
+        steps:
+          - script: |
+              set -Eeuo pipefail
+
+              docker-compose build --pull
+              docker-compose run --rm tests
+            displayName: 'Build & Run Tests'
+
+          - task: PublishTestResults@2
```

Comment:
vyhodil bych, stejně to nefunguje:

![image](https://user-images.githubusercontent.com/4319320/132842545-c4737e6e-9e9c-456f-8b2b-12f42670f76b.png)